test/test.h

```diff
@@ -206,6 +206,15 @@ __attribute__((noinline)) T not_constant(T x) {
   return x;
 }
 
+// This type generates compiler errors if it is copied.
+struct no_copy {
```

Comment:
Heh, this showed up last in the review and I was confused what "no_copy" means.

I think the standard term is "move-only".